platform/parser/api/src/main/java/org/codice/ddf/parser/Parser.java

```diff
@@ -21,9 +21,9 @@
 
 /**
  * Defines a service interface for converting between Objects and external representations of them.
- * Objects can be represented in myriad ways for transport and interoperability with other systems;
- * this service is intended to isolate some of the more particular needs of various converision
- * libraries.
+ * Objects can be represented in a myriad of ways for transport and interoperability with other
```

Comment:
"Myriad" is both a noun _and_ an adjective. I'm confident the original author of this comment knew what they meant. (They likely did _not_ intend the typo on "conversion", however.)